Elsewhen/Views/SettingsComponents/TimeListSettings.swift

```diff
@@ -8,13 +8,17 @@
 import SwiftUI
 
 struct TimeListSettings: View {
+    @EnvironmentObject internal var timeZoneGroupController: MykeModeTimeZoneGroupsController
```

Comment:
`internal`?

Reply:
Hmm, every other @EnvironmentObject in the codebase has it?

Reply:
Changed it to private, but left the others as they are because:

> Property 'orientationObserver' must be declared internal because it matches a requirement in internal protocol 'OrientationObserving'

Reply:
Ah, that might be fixable but for now I'm fine with leaving it.